#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_rm_clean.dpatch by  <picca@synchrotron-soleil.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove the lib/clean script

@DPATCH@
diff -urNad lisaac~/lib/clean lisaac/lib/clean
--- lisaac~/lib/clean	2007-09-24 00:20:01.000000000 +0200
+++ lisaac/lib/clean	1970-01-01 01:00:00.000000000 +0100
@@ -1,3 +0,0 @@
-for i in `find -name "*~"` ; do rm $i ; done
-for i in `find -name "*.c"` ; do rm $i ; done
-for i in `find -name "*.old"` ; do rm $i ; done
